lib/redmine_tags/patches/queries_helper_patch.rb

```diff
@@ -27,7 +27,7 @@ def column_content_with_redmine_tags(column, issue)
         def csv_content_with_redmine_tags(column, issue)
           value = column.value_object(issue)
 
-          if column.name == 'tags'
```

Comment:
I'm not in favour of using eql? instead of ==.

What about? 
```
if column.name == :tags
```

Reply:
Totally agree. `#eql?` is for Hash key equality, and definitely *never* should be used for comparison like this. Should be what @marius-balteanu said.

Reply:
If `column.name` by some reason sometimes returns string and sometimes symbol, just coerce it to string:

``` ruby
if column.name.to_s == "tags"
```

or to symbol:

``` ruby
if column.name.to_sym == :tags
```

or simply check both:

``` ruby
if column.name == :tags || coumn.name == "tags"
```